### PR TITLE
Feature 45 attach picutres to products

### PR DESCRIPTION
**Pull request by dimitrov93**

### CODE REVIEW

File: src/components/Button/PictureButton.vue [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      v-model="chosenFile"
+      accept=".jpeg, .jpg, .png"
+      @change="emitPicture"
+    ></v-file-input>
+  </v-container>
+</template>
+<script setup>
+import {ref} from 'vue'
+const chosenFile = ref(null)
+const props = defineProps({isDisabled: Boolean})
```
Comment (VladoKat) [inline src/components/Button/PictureButton.vue:16]: Are we using isDisabled? If not, let's remove it

File: src/views/ProductsDetails.vue [hunk 2]
```diff
@@ -198,25 +201,57 @@ const submitProduct = async () => {
...
 
+const handlePictureSelected = (chosenFile) => {
+  selectedPicture.value = chosenFile;
+};
+
+const isPictureValidated = () => {
+  if (selectedPicture.value <= 0) {
+    return false;
+  }
+  return true;
```
Comment (VladoKat) [inline src/views/ProductsDetails.vue:221]: Does this work in this case?
```suggestion
  return !!selectedPicture.value
```

Comment (VladoKat) [reply src/views/ProductsDetails.vue:221]: should be the same as the if though
```suggestion
  return selectedPicture.value > 0;
```

File: src/views/ProductsDetails.vue [hunk 3]
```diff
@@ -198,25 +201,57 @@ const submitProduct = async () => {
...
+      "Successfully added product and picture!"
+    );
+  } catch (error) {
+    snackbarProvider.showErrorSnackbar(
+      "Couldn't add the picture to the product!"
+    );
+  }
+};
+
+async function submitPicture(productResponse, selectedImage) {
```
Comment (VladoKat) [inline src/views/ProductsDetails.vue:237]: You can remove `selectedImage` and use `selectedPicture.value` instead.

**Review by VladoKat — CHANGES_REQUESTED** [review]
Overall looks good

File: src/components/Button/PictureButton.vue (hunk 1 above)
Comment (dimitrov93) [reply src/components/Button/PictureButton.vue:16]: done

File: src/views/ProductsDetails.vue (hunk 2 above)
Comment (dimitrov93) [reply src/views/ProductsDetails.vue]: done

File: src/views/ProductsDetails.vue (hunk 3 above)
Comment (dimitrov93) [reply src/views/ProductsDetails.vue]: done

**Review by VladoKat — APPROVED** [review]

**Pull request merged**